provider/clusterrosaclassic/cluster_rosa_classic_resource.go

```diff
@@ -397,12 +399,13 @@ func (r *ClusterRosaClassicResource) Schema(ctx context.Context, req resource.Sc
 				Attributes: map[string]schema.Attribute{
 					"username": schema.StringAttribute{
```

Comment:
So is it expected that the user never provides the username?
Why not allow this? I see that the code below allows the case in which the user add their own username.

Other thought here, it doesn't look so intuitive that user need to add empty list in order to trigger the admin credentials, maybe better also to add an boolean as well?

Reply:
@nirarg 
thanks for the review.
Yes, we should support user-provided username, and if not provided, "cluster-admin" is used.
For that logic, what's the correct configuration for this attribute? Is it ok as below?

```
					"username": schema.StringAttribute{
						Description: "Admin username that will be created with the cluster.",
						Default:    "cluster-admin",
						Validators:  identityprovider.HTPasswdUsernameValidators,
					},
```